docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -0,0 +1,395 @@
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/mp_serial_2_protocol.png
+    :align: center
+    :width: 50%
+    :alt: Setting SERIAL2_PROTOCOL to ESC Telemetry
+
+    Setting SERIAL2_PROTOCOL to ESC Telemetry
+
+The SERIAL2_BAUD parameter should also be set to match the baudrate of the Vertiq module. By default Vertiq modules use a baudrate of 115200, so
+set SERIAL2_BAUD to 115 as shown below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/mp_serial_2_baud.png
+    :align: center
+    :width: 50%
+    :alt: Setting SERIAL2_BAUD for 115200 Baudrate
+
+    Setting SERIAL2_BAUD for 115200 Baudrate
+
+The `ArduCopter documentation on setting up telemetry for BLHeli ESCs <hhttps://ardupilot.org/copter/docs/common-blheli32-passthru.html#esc-telemetry>`_
+mentions configuring SERVO_BLH_TRATE and SERVO_BLH_POLES. These can be left at their defaults for a basic telemetry test setup. If you want a
+higher telemetry update rate, you can change the SERVO_BLH_TRATE. For details on the effect of SERVO_BLH_POLES, see the :ref:`arducopter_erpm_vs_rpm`
+section below.
+
+After changing these parameters, reboot your flight controller.
+
+Testing Telemetry
+##################
+Live ESC telemetry updates can be viewed in Mission Planner under the Status section of the DATA tab. After properly configuring the flight controller,
+power on your connected Vertiq module. Select the Status section in the DATA tab, and scroll over to the esc1 section. The "esc1_curr", "esc1_rpm",
+"esc1_temp", and "esc1_volt" entries should update with live telemetry information, as shown below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/mp_no_spin_telemetry.png
+    :align: center
+    :width: 50%
+    :alt: Live Telemetry Updates in Mission Planner
+
+    Live Telemetry Updates in Mission Planner
+
+To test telemetry when spinning, use the Motor Test tab to spin the module. The image below shows an example of spinning the module at 5% throttle for 200 seconds,
+which will keep the module spinning for an extended period, making it easier to check the telemetry.
+
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/mp_motor_test.png
+    :align: center
+    :width: 50%
+    :alt: Motor Test in Mission Planner
+
+    Motor Test in Mission Planner to Test Telemetry When Spinning
+
+The image below shows telemetry data for a spinning module. Note the non-zero RPM value. For more information on how the flight controller calculates
+the displayed RPM value based on the value reported by the module, see the :ref:`arducopter_erpm_vs_rpm` section below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/mp_spin_telemetry.png
+    :align: center
+    :width: 50%
+    :alt: Live Telemetry Updates in Mission Planner When Spinning
+
+    Live Telemetry Updates in Mission Planner When Spinning
+
+.. _arducopter_erpm_vs_rpm:
+
+ERPM vs. RPM

```

Comment:
As I describe below, the flight controllers expect ERPM like KISS specifies, so by default its confused by what we send. For PX4 you can get around by setting a high number of motor poles, but for ArduCopter you're out of luck. Until we either change something on our end or on the ArduCopter end, the RPM value recorded by the flight controller will always be wrong, and anyone looking at telemetry logs will need to do a conversion.

Asana task for better integration: https://app.asana.com/0/1158790994115475/1204552170243362/f